### ISSUE TITLE
Potential Conflict with wp-cron.php in a Headless Setup

### ISSUE THREAD

**Issue by alexookah** [wpscholar-wp-plugins/simple-website-redirect#28]
I suspect this plugin might be blocking WordPress cron jobs (wp-cron.php) from running correctly. Here’s the context:

My WordPress installation is configured as a headless setup, so the front-end is rarely visited.
Recently, I noticed that scheduled tasks like backups and plugin auto-updates have stopped working.
I’m wondering if the plugin is redirecting requests to /wp-cron.php, which could explain why these cron jobs aren't being triggered.

Maybe this path should be added in the [plugin exluded paths](https://github.com/wpscholar-wp-plugins/simple-website-redirect/blob/21b64fd50f55c2af5c39a902399b0833ea5695ea/SimpleWebsiteRedirect.php#L263-L269)?

**Comment by wpscholar** [comment]
@alexookah Have you tried adding that path to the excluded paths via the UI (under advanced settings)? If that fixes it, then I'll definitely get that updated in the code. Thanks for the heads up.

**Comment by alexookah** [comment]
Yes, I just verified it. I added the path, and now cron jobs, such as plugin updates and scheduled backups, are working properly. The path to wp-cron.php is essential for cron jobs like automatic plugin updates and scheduled backups. I believe it should be included in the core's list of excluded paths.

**Comment by alexookah** [comment]
Created a PR for this
https://github.com/wpscholar-wp-plugins/simple-website-redirect/pull/29

**Comment by wpscholar** [comment]
Thanks for the PR @alexookah
I've merged it and will release a new version of the plugin soon.
